conf/supervisord-dev.conf

```diff
@@ -24,11 +23,20 @@ stderr_events_enabled=true
 stopsignal = KILL
 stopasgroup = true
 
-
 [eventlistener:logger]
 command=bin/logger --dev
 buffer_size=100
 events=PROCESS_LOG
 stderr_logfile=/dev/fd/1
 stderr_logfile_maxbytes=0
 stdout_logfile=/dev/null
+
+[unix_http_server]
+file = .supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl = unix://.supervisor.sock
+prompt = h-periodic
```

Comment:
A bunch of stuff that our other projects have but h-periodic didn't, and that the cookiecutter would add if it was templating this file. This stuff is used by `supervisorctl` and the Supervisor web interface.